### PR TITLE
Ignore build queue unit test

### PR DESCRIPTION
**Pull request by christ66**
Currently this unit test has a race condition when performing a release and is currently preventing a new release. Unit tests work when running with `mvn test`, and `mvn install`, however fail consistently when a release is performed.

@reviewbybees

### CODE REVIEW

**Comment by christ66** [comment]
@jglick noticed this first: https://github.com/jenkinsci/support-core-plugin/pull/14#discussion_r29801137.

File: src/test/java/com/cloudbees/jenkins/support/impl/BuildQueueTest.java [hunk 1]
```diff
@@ -51,6 +52,7 @@
 public class BuildQueueTest {
   @Rule public JenkinsRule j = new JenkinsRule();
 
+  @Ignore
```
Comment (jtnord) [inline src/test/java/com/cloudbees/jenkins/support/impl/BuildQueueTest.java]: always add a reason when you @Ignore
e.g. `@Ignore("Fails under release and I need to make one even though this test fails and is telling me something is broken, I think I know better");`

Comment (christ66) [inline src/test/java/com/cloudbees/jenkins/support/impl/BuildQueueTest.java]: Ah, thanks! Adding comment.

**Comment by jtnord** [comment]
:bug: on the lack of comment for @Ignore.
I would like to see the failure commented somewhere (either in this PR or a gist link).

If there is a race condition would it not be better to fix the code that causes it or fix the unit test?  This test presumably has been working before now, and as such the fact that something is failing should be a blocker for release.

**Comment by stephenc** [comment]
I suspect the race condition will be resolved once core is 1.607+

**Comment by stephenc** [comment]
:bug: for `@Ignore` without comment

**Comment by stephenc** [comment]
:bee:

**Comment by stephenc** [comment]
would be even better if you had a `// TODO enable once Jenkins 1.607+` but I'll not hold up for that request

**Comment by oleg-nenashev** [comment]
:bee:

**Comment by jtnord** [comment]
:bee:

**Pull request merged**